web/templates/job/new.html.eex

```diff
@@ -5,7 +5,10 @@
     you'll be sent an email asking if you'd like to extend (for&nbsp;free).</p>
     <p>Your position should be near Raleigh, NC, and related to UX, UI, or
     visual design for applications or the&nbsp;web.</p>
-    <%= link("← Back to Job Board", to: job_path(@conn, :index), class: "back-link") %>
+    <%= link("← Back to Job Board", to:
```

Comment:
How about bringing the `to:` down on a new line and putting the closing `) %>` on is own line?